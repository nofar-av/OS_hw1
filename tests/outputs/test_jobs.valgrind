==28114== Memcheck, a memory error detector
==28114== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28114== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28114== Command: /home/omer/OS_wet1/smash
==28114== Parent PID: 28111
==28114== 
==28114== 
==28114== FILE DESCRIPTORS: 4 open at exit.
==28114== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_jobs.valgrind
==28114==    <inherited from parent>
==28114== 
==28114== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_jobs.err
==28114==    <inherited from parent>
==28114== 
==28114== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_jobs.out
==28114==    <inherited from parent>
==28114== 
==28114== Open file descriptor 0:
==28114==    <inherited from parent>
==28114== 
==28114== 
==28114== HEAP SUMMARY:
==28114==     in use at exit: 0 bytes in 0 blocks
==28114==   total heap usage: 27 allocs, 27 frees, 82,184 bytes allocated
==28114== 
==28114== All heap blocks were freed -- no leaks are possible
==28114== 
==28114== For counts of detected and suppressed errors, rerun with: -v
==28114== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
