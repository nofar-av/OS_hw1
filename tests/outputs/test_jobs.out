smash> smash> smash> [1] sleep 100& : 5630 0 secs
[2] sleep 200& : 5632 0 secs
smash> 